sdk/python/kfp/dsl/_container_op.py

```diff
@@ -63,24 +59,25 @@ def __init__(self, name: str, image: str, command: str=None, arguments: str=None
     self.pod_labels = {}
 
     matches = []
+    to_search = []
     if arguments:
-      for arg in arguments:
-        match = re.findall(r'{{pipelineparam:op=([\w-]*);name=([\w-]+);value=(.*?)}}', str(arg))
-        matches += match
+      to_search += arguments
+    if command:
+      to_search += command
+
+    for arg in to_search:
```

Comment:
for arg in (command or []) + (arguments or []):

Reply:
Good idea. PTAL